content/2.application.md

```diff
@@ -42,10 +42,13 @@ The Participant node is a node in the Corda business network with the Payments C
 ### Payments CorDapp
 
 The Payments CorDapp is a custom CorDapp developed by the Participant or the BNO using the Gevamu Payments SDK. 
+In order to connect the CorDapp with the Gevamu gateway associated with the PSP, the developer must include details of the PSP.
```

Comment:
Developer shouldn't include PSP information. Maybe you are trying to say that every payment initiation request requires specific gateway to be specified.

Reply:
I believe that the requested change is not related to this section at all. It's about Gevamu Payment Gateway, not the CorDapp

Reply:
@Hdd87 @d0rich Let's remove it from this section